pkg/controller/compliancescan/scan.go

```diff
@@ -27,6 +32,12 @@ func (r *ReconcileComplianceScan) createScanPods(instance *compv1alpha1.Complian
 			return err
 		}
 
+		if instance.Spec.TailoringConfigMap != nil {
+			if err := r.addTailoringVolume(instance, pod); err != nil {
+				return err
+			}
+		}
+
```

Comment:
Why isn't this hunk part of the function that creates the pod structure in the first place?

Reply:
just thought that having it in one spot would make validations and addition of the volume easier.

Reply:
It's not a big deal.